src/clarity-angular/tabs/_tabs.clarity.scss

```diff
@@ -5,6 +5,12 @@
 @import "../utils/mixins.clarity";
 
 @include exports('tabs.clarity') {
+  @media screen {
+    section[aria-hidden="true"] {
+      display: none;
+    }
+  }
+
   [data-hidden="true"] {
```

Comment:
I don't know if we need data-hidden. I removed it from the new wizard. Maybe we should just deprecate it and remove it in 0.9?